lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -47,6 +58,15 @@ const NUMERICAL_EXPECTATION_REGEXP =
  * @return {boolean}
  */
 function matchesExpectation(actual, expected) {
+  // Skip if not using the minimum necessary Chrome.
+  if (expected && expected._chromeMajorVersion) {
```

Comment:
🚲 🏠 

`minChromeVersion`
`minChromeMajorVersion`
`minChromeMilestone`

any takers?

Reply:
> 🚲 🏠

something with min in it definitely seems good